Rakefile

```diff
@@ -1,7 +1,7 @@
 require "logstash/devutils/rake"
 
 desc "Get jars"
-task :get_jars do
+task :vendor do
```

Comment:
It was used for `file-dependencies` and the vendor.json, maybe this code could be refactored to use it.